model/trace/rntbd.yaml

```diff
@@ -0,0 +1,28 @@
+groups:
+  - id: rntbd
+    type: span
+    span_kind: client
+    brief: >
+      Network-level attributes for Cosmos DB. It is Microsoft proprietry protocol for Cosmos DB.
+    attributes:
+      - ref: rntbd.url
+        tag: network-level-tech-specific
+        requirement_level: required
+
+      - ref: rntbd.status_code
+        tag: network-level-tech-specific
+        requirement_level: required
+
+      - ref: rntbd.sub_status_code
+        tag: network-level-tech-specific
+        requirement_level: required
```

Comment:
is  it always available?

Reply:
what if you don't get any response? (timeout)

we use `error.type` attribute to record errors in addition to domain-specific status codes. This way users don't need to know a list of all success codes to understand what's qualified as success.